### PR TITLE
form in progress

### PR DESCRIPTION
**Pull request by Kaalaayaa**

### CODE REVIEW

**Review by mariagluck — APPROVED** [review]

File: src/components/ListingPage/Form.jsx [hunk 1]
```diff
@@ -10,32 +10,121 @@ export default function Form(){
     return(
         <div className="form">
             <h1>CREATE LISTING</h1>
-            <form onSubmit={handleSubmit(onSubmit)}>
+            <form enctype="multipart/form-data" onSubmit={handleSubmit(onSubmit)}>
```
Comment (Mxx1029) [inline src/components/ListingPage/Form.jsx:13]: could you check, if in React we also need to give a `method="post" here?

File: src/components/ListingPage/Form.jsx [hunk 2]
```diff
@@ -50,36 +139,54 @@ export default function Form(){
...
                     <label htmlFor="">Website</label>
+                    <input type="text" {...register("Website", {min: 5})} />
                 </div>
                 <div>
                     <label htmlFor="">Email</label>
+                    <input type="text" {...register("Email", {required: true, pattern: /^\S+@\S+$/i})} />
                 </div>
                 <div>
                     <label htmlFor="">Phone</label>
+                    <input type="tel" {...register("Phone", {required: true, minLength: 6, maxLength: 12})} />
```
Comment (Mxx1029) [inline src/components/ListingPage/Form.jsx:172]: maxLength: 15, maybe ? making 004915151774246 possible (which is my mobile number which already has one additional number in the provider part of the number)

File: src/components/ListingPage/Form.jsx [hunk 3]
```diff
@@ -50,36 +139,54 @@ export default function Form(){
...
                 </div>
                 <div>
                     <label htmlFor="">Instagram</label>
+                    <input type="text" {...register("Instagram", {min: 3})} />
                 </div>
                 <div>
```
Comment (Mxx1029) [inline src/components/ListingPage/Form.jsx:178]: did you throw out twitter? if so, i need to throw it out of the model as well

**Review by Mxx1029 — COMMENTED** [review]
Awesome work! 💯 
 just 3 tiny things, see in comments

**Pull request merged**